server/api/controllers/tripController.js

```diff
@@ -22,47 +22,55 @@ class TripController {
 		});
 	}
 
-	static getAllTrips(req, res) {
+	static async getAllTrips(req, res) {
 		const allTrips = Trip.getAllTrips();
-		if (!allTrips.length) {
-			return res.status(404).json({
-				status: 'error',
-				error: 'Not found',
+		if (await allTrips) {
+			return res.status(200).json({
+				status: '200',
+				message: 'success',
+				data: await allTrips,
 			});
 		}
-		return res.status(200).json({
-			status: 'success',
-			data: allTrips,
+		return res.status(404).json({
+			status: 'error',
+			error: 'Not found',
 		});
 	}
 
-	static getSpecificTrip(req, res) {
+	static async getSpecificTrip(req, res) {
 		const id = parseInt(req.params.id);
+		// eslint-disable-next-line prefer-destructuring
+		// eslint-disable-next-line no-unused-vars
 		const specificTrip = Trip.getSpecificTrip(id);
+		if (await specificTrip) {
+			return res.status(200).json({
+				status: 'success',
+				data: await specificTrip,
+			});
+		}
 		if (!specificTrip) {
 			return res.status(404).json({
 				status: 'error',
 				error: 'Not found',
 			});
 		}
-		return res.status(200).json({
-			status: 'success',
-			data: specificTrip,
-		});
 	}
 
-	static cancelTrip(req, res) {
+	static async cancelTrip(req, res) {
 		const id = parseInt(req.params.id);
-		const cancelTrip = Trip.cancelTrip(id);
-		if (!cancelTrip) {
-			return res.status(404).json({
-				status: 'error',
-				error: 'Not found',
+		// eslint-disable-next-line prefer-destructuring
```

Comment:
Destruct and remove this inline eslint rule

Reply:
Removed